ssclj/jar/test/com/sixsq/slipstream/ssclj/migrate/user_cred_test.clj

```diff
@@ -0,0 +1,262 @@
+             :VALUE                 "20",
+             :MANDATORY             true,
+             :CATEGORY              "exoscale-ch-gva",
+             :ORDER_                0,
+             :INSTRUCTIONS          nil,
+             :DESCRIPTION           "Number of VMs the user can start for this cloud",
+             :CONTAINER_RESOURCEURI user2,
+             :TYPE                  0,
+             :READONLY              true,
+             :ENUMVALUES            nil,
+             :NAME                  "exoscale-ch-gva.quota.vm"}
+            {:ID                    73487,
+             :VALUE                 "",
+             :MANDATORY             true,
+             :CATEGORY              "exoscale-ch-gva",
+             :ORDER_                10,
+             :INSTRUCTIONS          "If you use the DNS service of Exoscale you can set here the name of one of your domains and SlipStream will create a subdomain for each of your instances",
+             :DESCRIPTION           "Name of your domain managed by Exoscale",
+             :CONTAINER_RESOURCEURI user2,
+             :TYPE                  0,
+             :READONLY              false,
+             :ENUMVALUES            nil,
+             :NAME                  "exoscale-ch-gva.domain.name"}
+            {:ID                    76693,
+             :VALUE                 "secret",
+             :MANDATORY             true,
+             :CATEGORY              "exoscale-ch-gva",
+             :ORDER_                20,
+             :INSTRUCTIONS          "On the Exoscale web interface you can find this information on <code>Account > Profile > API Keys</code>",
+             :DESCRIPTION           "Secret Key",
+             :CONTAINER_RESOURCEURI user2,
+             :TYPE                  2,
+             :READONLY              false,
+             :ENUMVALUES            nil,
+             :NAME                  "exoscale-ch-gva.password"}]})
+
+
+(deftest extracting-data
+  (let [category-exo-gva "exoscale-ch-gva"
+        category-otc "open-telekom-de1"
+        wrong-category "wrong"
+        expected1 {:credentialTemplate {:href        "credential-template/store-cloud-cred-exoscale"
+                                        :key         "EXO0000000001"
+                                        :secret      "secret"
+                                        :connector   "connector/exoscale-ch-gva"
+                                        :domain-name ""}
+                   :user               user1}
+        expected2 {:credentialTemplate {:href        "credential-template/store-cloud-cred-exoscale"
+                                        :key         "EXO0000000002"
+                                        :secret      "secret"
+                                        :connector   "connector/exoscale-ch-gva"
+                                        :domain-name ""}
+                   :user               user2}]
+    (is (= nil (t/extract-data nil nil nil)))
```

Comment:
Please use `are` to remove the repetition in this test.  Also you can replace `(= (nil ...))` with just `(nil? ...)`.